cfme/tests/infrastructure/test_rest_automation_request.py

```diff
@@ -257,6 +258,23 @@ def test_edit_requests(self, collection, appliance, requests_pending, from_detai
         # testing BZ 1418338
         edit_requests(collection, appliance.rest_api, requests_pending, from_detail)
 
+    @pytest.mark.tier(1)
+    @pytest.mark.meta(automates=[BZ(1723830)])
+    def test_multiple_automation_requests(self, collection, appliance, vm):
+        """
+        Polarion:
+            assignee: ghubale
+            initialEstimate: 1/30h
+            casecomponent: Automate
+
+        Bugzilla:
+            1723830
+        """
+        requests_data = _automation_requests_data(vm, approve=False, num=2000)
```

Comment:
@ganeshhubale I see that the BZ was written when a customer submitted 60000 requests, is 2000 enough? Thoughts on parametrizing to more? e.g. `[2000, 10,000, 30,000, 60,000]`

Reply:
Thanks @john-dupuy for going through BZ.
- According to me, parametrizing this test case with different number of requests is not actually a use case. Here we want to check if huge number of automation requests provisioned are getting denied or not.
- Yes, we can increase the number of requests up to the number we want. So I updated it to 60,000 and making this PR long_running(Although this is rest test case)

Reply:
Restarting PRT with `--long-running` included in the options, as this change caused the test to not be collected in run 33445